app/src/main/java/org/mozilla/fenix/settings/address/AddressEditorFragment.kt

```diff
@@ -100,4 +106,12 @@ class AddressEditorFragment : SecureFragment(R.layout.fragment_address_editor) {
         }
         else -> false
     }
+
+    companion object {
+        val defaultCountry = AddressCountry("US", "UNITED STATES")
+        val availableCountries = mapOf(
```

Comment:
nit: should these be all-caps?

Reply:
I think caps might usually be reserved for `const` vals, which these technically are not. I was following the example we had in `WallpaperManager`. I'm not seeing a ton of consistency in a short search of other `companion object`s. I think the caps casing is also a holdout from a stricter `static` variable convention in Java. I'm not personally a huge fan of it, but I'm happy to switch it if you'd prefer.